greensnow-feed/info.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "greensnow-feed",
+  "version": "1.0.0",
+  "label": "GreenSnow Feed",
+  "description": "GreenSnow is a team consisting of the best specialists in computer security, we harvest a large number of IPs from different computers located around the world. This connector facilitates automated operations such as indicators.<br/><br/>This connector has a dependency on the <a href=\"/content-hub/all-content/?contentType=solutionpack&amp;tag=ThreatIntelManagement\" target=\"_blank\" rel=\"noopener\">Threat Intel Management Solution Pack</a>. Install the Solution Pack before enabling ingestion of Threat Feeds from this source.",
```

Comment:
HTML tags in description 
Is it rendered properly??

Reply:
Yes, it renders properly.